example/distributed-training/log/cifar10/incept_4

```diff
@@ -0,0 +1,600 @@
+INFO:root:Epoch[20] Train-accuracy=0.931471
+INFO:root:Epoch[20] Time cost=10.662
+INFO:root:Epoch[20] Validation-accuracy=0.866486
+INFO:root:Epoch[20] Validation-accuracy=0.863381
+INFO:root:Epoch[20] Validation-accuracy=0.863281
+INFO:root:Epoch[20] Validation-accuracy=0.860477
+INFO:root:Epoch[20] Validation-accuracy=0.866687
+INFO:root:Epoch[21] Train-accuracy=0.936906
+INFO:root:Epoch[21] Time cost=10.774
+INFO:root:Epoch[21] Train-accuracy=0.938349
+INFO:root:Epoch[21] Time cost=10.720
+INFO:root:Epoch[21] Train-accuracy=0.938264
+INFO:root:Epoch[21] Time cost=10.604
+INFO:root:Epoch[21] Train-accuracy=0.937585
+INFO:root:Epoch[21] Time cost=10.646
+INFO:root:Epoch[21] Train-accuracy=0.938604
+INFO:root:Epoch[21] Time cost=10.663
+INFO:root:Epoch[21] Validation-accuracy=0.874900
+INFO:root:Epoch[21] Validation-accuracy=0.877404
+INFO:root:Epoch[21] Validation-accuracy=0.871795
+INFO:root:Epoch[21] Validation-accuracy=0.873598
+INFO:root:Epoch[21] Validation-accuracy=0.875801
+INFO:root:Epoch[22] Train-accuracy=0.943190
+INFO:root:Epoch[22] Time cost=10.764
+INFO:root:Epoch[22] Train-accuracy=0.941406
+INFO:root:Epoch[22] Time cost=10.711
+INFO:root:Epoch[22] Train-accuracy=0.943190
+INFO:root:Epoch[22] Time cost=10.661
+INFO:root:Epoch[22] Train-accuracy=0.939963
+INFO:root:Epoch[22] Time cost=10.646
+INFO:root:Epoch[22] Train-accuracy=0.938349
+INFO:root:Epoch[22] Time cost=10.647
+INFO:root:Epoch[22] Validation-accuracy=0.867488
+INFO:root:Epoch[22] Validation-accuracy=0.870994
+INFO:root:Epoch[22] Validation-accuracy=0.873197
+INFO:root:Epoch[22] Validation-accuracy=0.871695
+INFO:root:Epoch[22] Validation-accuracy=0.873498
+INFO:root:Epoch[23] Train-accuracy=0.946332
+INFO:root:Epoch[23] Time cost=10.705
+INFO:root:Epoch[23] Train-accuracy=0.945312
+INFO:root:Epoch[23] Time cost=10.782
+INFO:root:Epoch[23] Train-accuracy=0.947520
+INFO:root:Epoch[23] Time cost=10.617
+INFO:root:Epoch[23] Train-accuracy=0.948115
+INFO:root:Epoch[23] Time cost=10.661
+INFO:root:Epoch[23] Train-accuracy=0.946841
+INFO:root:Epoch[23] Time cost=10.660
+INFO:root:Epoch[23] Validation-accuracy=0.876202
+INFO:root:Epoch[23] Validation-accuracy=0.868490
+INFO:root:Epoch[23] Validation-accuracy=0.877804
+INFO:root:Epoch[23] Validation-accuracy=0.872496
+INFO:root:Epoch[23] Validation-accuracy=0.880208
+INFO:root:Epoch[24] Train-accuracy=0.950068
+INFO:root:Epoch[24] Time cost=10.726
+INFO:root:Epoch[24] Train-accuracy=0.951087
+INFO:root:Epoch[24] Time cost=10.784
+INFO:root:Epoch[24] Train-accuracy=0.949389
+INFO:root:Epoch[24] Time cost=10.658
+INFO:root:Epoch[24] Train-accuracy=0.946841
+INFO:root:Epoch[24] Train-accuracy=0.949389
+INFO:root:Epoch[24] Time cost=10.664
+INFO:root:Epoch[24] Time cost=10.654
+INFO:root:Epoch[24] Validation-accuracy=0.868690
+INFO:root:Epoch[24] Validation-accuracy=0.863482
+INFO:root:Epoch[24] Validation-accuracy=0.867688
+INFO:root:Epoch[24] Validation-accuracy=0.864083
+INFO:root:Epoch[24] Validation-accuracy=0.865084
```

Comment:
No, they should not be synced during training, because they are only used for prediction. They only need to be synced after each data pass . It is even OK to sync only when we save model, c.f. https://github.com/dmlc/mxnet/blob/master/python/mxnet/model.py#L399